app/gui2/src/stores/suggestionDatabase/entry.ts

```diff
@@ -62,7 +62,7 @@ export interface SuggestionEntry {
   /// A name of a custom icon to use when displaying the entry.
   iconName?: string
   /// An index of a group from group list in suggestionDb store this entry belongs to.
-  groupIndex?: number
+  groupIndex?: number | undefined
```

Comment:
Is it really needed?

Reply:
i think it's no longer needed, after the latest merge